### PR TITLE
KILT types definition repo

### PR DESCRIPTION
**Pull request by Dudleyneedham**
Adding KILT types definition repo.

### CODE REVIEW

File: packages/apps-config/package.json [hunk 1]
```diff
@@ -14,6 +14,7 @@
     "@edgeware/node-types": "^3.3.4",
     "@equilab/definitions": "1.0.3",
     "@interlay/polkabtc-types": "^0.6.2",
+    "@kiltprotocol/type-definitions": "latest",
```
Comment (jacogr) [inline packages/apps-config/package.json:17]: This is great. Could you rather point to the version - the reason being is that I do update the dependencies (weekly, sometimes more) and `yarn upgrade-interactive` I can see what got bumped. It is unlikely that a `rm -rf yarn.lock` is called.

**Review by jacogr — COMMENTED** [review]
One small comment for my sanity. Love these, makes it much easier from both sides.

File: packages/apps-config/package.json (hunk 1 above)
Comment (Dudleyneedham) [reply packages/apps-config/package.json]: Sure! No problem. I was wondering which would be preferred. 👍  I have updated it now.

**Comment by Dudleyneedham** [comment]
I agree. It is something we have been meaning to do for a while.

**Pull request merged**